tools/golang-json-rpc-tests/.env.example

```diff
@@ -0,0 +1,4 @@
+# The operator private keys start with "0x" and are in hexadecimal format
+# Your account alias ECDSA hex-encoded private key (Ex: 0xb46751179bc8aa9e129d34463e46cd...)
+OPERATOR_PRIVATE_KEY=0x105d050185ccb907fba04dd92d8de9e32c18305e097ab41dadda21489a211524
```

Comment:
Q: where is this value from?
Better to leave it empty or as the text that existed before

Reply:
@Nana-EC 
It's coming from the local node and it's used the same way here:
- https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/tools/hardhat-example/.env.example
- https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/tools/truffle-example/.env.example
- https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/tools/subgraph-example/.env.example
- https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/tools/web3js-example/.env.example

I tried to be consistent.

Reply:
Okay great.
I like there to be a comment so people know where it's coming from. Just a link to local node in the comments is sufficient

Reply:
@Nana-EC Comment adjusted
@quiet-node Yes, those values are utilized in CI (please refer to https://github.com/hashgraph/hedera-json-rpc-relay/pull/2679#discussion_r1675406058)